docs/htdocs/info/docs/tools/vep/vep_formats.html

```diff
@@ -574,7 +574,7 @@ <h3 style="margin:20px 0px 30px">Other output fields:</h3>
         <li><b>INTRON</b> - the intron number (out of total number)</li>
         <li><b>EXON</b> - the exon number (out of total number)</li>
         <li><b>DOMAINS</b> - the source and identifer of any overlapping protein domains</li>
-        <li><b>DISTANCE</b> - Shortest distance from variant to transcript</li>
+        <li><b>DISTANCE</b> - Shortest distance from variant to transcript. <i>Note: DISTANCE=0 is possible for insertions happening just before or after a transcript because variant co-ordinates is considered to be the flanking bases where insertion happens.</i></li>
```

Comment:
`variant co-ordinates is` -> `variant co-ordinates are`?

Also, I think `co-ordinates` -> `coordinates` (that's the way this word is spelled in other documentation files in this repository)

Reply:
Agree with @azangru 

Also can we change DISTANCE=0 to A DISTANCE of 0 is... (elsewhere in the doc we avoid using mathematical notation)

Reply:
Thanks @azangru , @jamie-m-a , updated the text.